### PR TITLE
To List in SessionManager

### PR DESCRIPTION
**Pull request by Aliyevvvg**
**When getting session;**

### CODE REVIEW

File: TCBApp/TCBApp/TelegramBot/SimpleSessionManager/SessionManager.cs
Comment (0605AbMu) [inline TCBApp/TCBApp/TelegramBot/SimpleSessionManager/SessionManager.cs:1]: Find old session before create new session and return it

**Review by 0605AbMu — CHANGES_REQUESTED** [review]
Fix all!

**Comment by Aliyevvvg** [comment]
Can you check

**Review by 0605AbMu — CHANGES_REQUESTED** [review]
Wrong logic options

**Comment by Aliyevvvg** [comment]
check it

**Review by 0605AbMu — CHANGES_REQUESTED** [review]
Wrong logic options

**Comment by Aliyevvvg** [comment]
**Check It**

**Review by 0605AbMu — APPROVED** [review]
Good job!

**Pull request merged**